flock_agent/gui/systray.py

```diff
@@ -7,10 +7,17 @@
 
 
 class SysTray(QtWidgets.QSystemTrayIcon):
+    show_clicked = QtCore.pyqtSignal()
+
     def __init__(self, common):
         super(SysTray, self).__init__()
         self.c = common
 
+        menu = QtWidgets.QMenu()
+        show_action = menu.addAction("Show Flock")
+        show_action.triggered.connect(lambda: self.show_clicked.emit())
```

Comment:
I'm curious, what happens if you just pass self.show_clicked.emit (without parens, and no lambda) here?